debug/app/bespoke/components/customComponent.js

```diff
@@ -0,0 +1,27 @@
+SMK.HANDLER.set('BespokeTool--101', 'activated', (smk, tool) => {

```

Comment:
Maybe include a comment indicating what is happening and/or how this works?
You could mention that a Bespoke tool was added to the app/smk-config and that the 'BespokeTool--101' is a concatenation of 'BespokeTool' and the instance name of the configured tool (ie. '101'). 'Activated' indicates that the handler will run when the tool is activated (versus the other options of initialized, deactivated and triggered. The third argument in the SMK.Handler.set() function is the anonymous function that executes when the BespokeTool--101 is activated.